src/main/java/io/confluent/connect/jdbc/sink/DbStructure.java

```diff
@@ -75,6 +75,9 @@ public boolean createOrAmendIfNecessary(
         } catch (SQLException e) {
           throw sqle;
         }
+      } catch (TableAlterOrCreateException te) {
+        log.warn(te.getMessage());
+        throw te;
```

Comment:
Maybe I missed something, how is this different from what was before, other than a warn msg? how would this help with the NPE issue.

Reply:
If we do not catch this exception here, it does not bubble up. Instead, we try to proceed with table creation and that results in a NPE

Reply:
Now that this is caught and thrown, it is handled gracefully and results in an error that either table is not already created or auto create is not configured.

Reply:
@ddasarathan explained to me via zoom. Now the code makes sense to me.